gym_sts/envs/base.py

```
@@ -297,6 +300,12 @@ def reset(
         self.sts_seed = sts_seed
 
         obs = self.communicator.start("DEFECT", 0, sts_seed)
+
+        # In my experience the game isn't actually stable here, and we have
+        # to wait for a bit before the game actually starts.
+        while obs.screen_type == "MAIN_MENU":
+            obs = self.observe()
```

Can we wait a short time between loops?

---

Also, can we raise an exception after some number of attempts, rather than looping forever?

---

Done.